server/sonar-web/src/main/js/apps/projectActivity/components/forms/AddGraphMetric.js

```diff
@@ -19,21 +19,20 @@
  */
 // @flow
 import React from 'react';
-import classNames from 'classnames';
-import Modal from '../../../../components/controls/Modal';
-import Select from '../../../../components/controls/Select';
-import Tooltip from '../../../../components/controls/Tooltip';
+import { find } from 'lodash';
+import AddGraphMetricPopup from './AddGraphMetricPopup';
+import DropdownIcon from '../../../../components/icons-components/DropdownIcon';
+import BubblePopup from '../../../../components/common/BubblePopup';
+import BubblePopupHelper from '../../../../components/common/BubblePopupHelper';
+import MultiSelect from '../../../../components/common/MultiSelect';
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import of 'MultiSelect'. ([javascript:S1128](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS1128))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&pullRequest=3149&issues=AWIovI3dnhKRZuqhdqYs&open=AWIovI3dnhKRZuqhdqYs)

[ISSUE_KEY]: AWIovI3dnhKRZuqhdqYs